### PR TITLE
Bettering things

### PR DESCRIPTION
**Pull request by F-fengzi**
No guarantee if it will work or not, please review the code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+*Custom WIM not recommended
 
 INSTRUCTIONS:
 
 1. Go to the releases and download "worli.sh"
 
 2. Put the script into a empty folder WITH NO SPACES IN ITS NAME!
 
-3. Open the terminal in that folder and run "sudo ./worli.sh" (or just ./worli.sh in a root shell)
+3. Open the terminal in that folder and run `sudo ./worli.sh` / `sudo bash worli.sh` (or just `./worli.sh` / `bash worli.sh` in a root shell)
```
Comment (buddyjojo) [inline README.md]: perhaps change the / to something less unixy

File: worli.sh [hunk 2]
```diff
@@ -1,266 +1,259 @@
...
+    ;;
+    [2])
+    export PI="3"
+    ;;
     *)
- echo "Invalid input..."
- exit 1
- ;;
-esac 
+    echo "Unsupported model"
```
Comment (buddyjojo) [inline worli.sh]: hm what if they formatted it incorrectly but said a supported pi?

File: worli.sh [hunk 3]
```diff
@@ -1,266 +1,259 @@
...
+    exit 1
+    ;;
+esac
 
 echo " "
-echo -e "do you want the tool to download the uefi, drivers and pe-installer or use your own files?"
+echo -e "Do you want the tool to download the UEFI, drivers, and PE-installer automatically? Say 'N' to use your own files"
```
Comment (buddyjojo) [inline worli.sh:35]: , and 
need i say more?

File: worli.sh [hunk 4]
```diff
@@ -1,266 +1,259 @@
...
+echo "Prerequisites"
 echo " "
-echo "- (NOT RECOMMENDED) if you want use use a custom wim rename it to 'install.wim' NOTE: you still need the iso the wim came from"
+echo "- Get the windows ISO: https://worproject.ml/guides/getting-windows-images"
+echo "- Rename the ISO file to 'win.iso'"
 echo " "
-echo "- If you're using a Raspberry Pi 4, you must update the bootloader to the latest version: https://www.raspberrypi.org/documentation/computers/raspberry-pi.html#updating-the-bootloader"
+echo "- (NOT RECOMMENDED) If you want use use a custom WIM, rename it to 'install.wim'. NOTE: You will still need the ISO where the WIM came from"
 echo " "
+echo "- If you're using a Raspberry Pi 4, you may have to update the Bootloader to the latest version: https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#updating-the-bootloader"
```
Comment (buddyjojo) [inline worli.sh]: hm mairo said must, perhaps its better to leave it like that?

File: worli.sh [hunk 5]
```diff
@@ -1,266 +1,259 @@
...
 then
-    echo "- parted package not installed. install it (for debian and ubuntu its 'sudo apt install parted', for arch it's also parted)"
+    echo "- 'parted' package not installed. Install it (For Debian and Ubuntu, run 'sudo apt install parted'; For Arch, run 'sudo pacman -S parted')"
     exit 1
 fi
-
 if ! command -v mkfs.ntfs &> /dev/null
 then
-    echo "- mkfs.ntfs command not found. install ntfs support somehow (like ntfs-3g and ntfsprogs)"
+    echo "- 'mkfs.ntfs' command not found. Install NTFW support somehow (such as ntfs-3g and ntfsprogs)"
```
Comment (buddyjojo) [inline worli.sh]: ntfw

File: worli.sh [hunk 6]
```diff
@@ -1,266 +1,259 @@
...
-read -r -p "[/dev/*] eg 'sdb', 'mmcblk0': " disk
-
-echo "you have selected '$disk' is this correct?"
+echo "You have selected '$disk', is this correct?"
 read -r -p "[Y/N]: " input
 case $input in
     [yY][eE][sS]|[yY])
- echo "ok $disk it is"
- ;;
+    echo "OK '$disk' it is then"
```
Comment (buddyjojo) [inline worli.sh]: idk caps ok is kinda boomer in my eyes dont ya think

File: worli.sh [hunk 7]
```diff
@@ -1,266 +1,259 @@
...
+    exit 1
+    ;;
+esac
 
 if [[ $disk == *"mmcblk"* ]]; then
     export nisk="${disk}p"
 else
     export nisk="$disk"
 fi
-
```
Comment (buddyjojo) [inline worli.sh:216]: i like to separate my code

File: worli.sh [hunk 8]
```diff
@@ -291,99 +284,94 @@ mount /dev/$nisk'1' /media/bootpart
...
 mount $iso /tmp/isomount
-
 cp -r /tmp/isomount/boot /media/bootpart
 cp -r /tmp/isomount/efi /media/bootpart
 mkdir /media/bootpart/sources
 cp /tmp/isomount/sources/boot.wim /media/bootpart/sources
 
 if [ -f $wim ]; then
     cp $wim /media/winpart
-else
```
Comment (buddyjojo) [inline worli.sh:308]: lmao

File: worli.sh [hunk 9]
```diff
@@ -291,99 +284,94 @@ mount /dev/$nisk'1' /media/bootpart
...
-else 
+else
     cp /tmp/isomount/sources/install.wim /media/winpart
 fi
 
 umount /tmp/isomount
 
 echo " "
-echo "copying the drivers to the drive..."
-echo " "
```
Comment (buddyjojo) [inline worli.sh:316]: perhaps the stuff below output stuff?

**Review by buddyjojo — COMMENTED** [review]
some suggestions

File: worli.sh (hunk 5 above)
Comment (buddyjojo) [reply worli.sh]: fuck you changed that

Comment (buddyjojo) [reply worli.sh]: sorry

File: worli.sh (hunk 3 above)
Comment (F-fengzi) [reply worli.sh:35]: Go learn English

File: worli.sh (hunk 4 above)
Comment (F-fengzi) [reply worli.sh]: aight then (tho later versions of rpi4 comes with the latest bootloader already, that's why I'm saying that)

File: worli.sh (hunk 6 above)
Comment (F-fengzi) [reply worli.sh]: ~~it is the only correct grammar~~ tho i agree, will change it back

File: worli.sh (hunk 5 above)
Comment (F-fengzi) [reply worli.sh]: should've typed nsfw

File: worli.sh (hunk 9 above)
Comment (F-fengzi) [reply worli.sh:316]: it is, check again with split view, might be more clear

**Comment by F-fengzi** [comment]
Remember when merging pull request, choose merge to one commit

File: worli.sh (hunk 9 above)
Comment (buddyjojo) [reply worli.sh:316]: no i mean the 'echo " "''
you changed 

echo " "
e
echo " "
command 

to 

echo " "
e
command
echo " "

File: worli.sh (hunk 3 above)
Comment (buddyjojo) [reply worli.sh:35]: i'm not arguing over a comma but i remember that not being a thing you can do

**Review by buddyjojo — COMMENTED** [review]
review of your reveiw

File: worli.sh (hunk 3 above)
Comment (F-fengzi) [reply worli.sh:35]: wait what

**Comment by buddyjojo** [comment]
> Remember when merging pull request, choose merge to one commit

what makes u say that 
nn- nothing happened yesterday

**Comment by F-fengzi** [comment]
F
~~i actually dunno~~

**Comment by buddyjojo** [comment]
~~i totally didn't mess up amogos~~

**Comment by F-fengzi** [comment]
~~yeah sure i totally didn’t just check~~

**Comment by F-fengzi** [comment]
pls reviw

**Comment by buddyjojo** [comment]
it has been combined with the code that i wrote thats hosted on this platform called github thats owned by microsoft and used the git protocol or whatever

**Comment by F-fengzi** [comment]
git version control system, member of the Software Freedom Conservancy, by Linus Torvalds

**Pull request merged**